cmd/goal/clerk.go

```diff
@@ -1097,7 +1094,7 @@ var dryrunCmd = &cobra.Command{
 			}
 			pass, err := logic.Eval(txn.Lsig.Logic, ep)
 			// TODO: optionally include `inspect` output here?
-			fmt.Fprintf(os.Stdout, "tx[%d] cost=%d trace:\n%s\n", i, cost, sb.String())
+			fmt.Fprintf(os.Stdout, "tx[%d] trace:\n%s\n", i, sb.String())
```

Comment:
why not printing the cost after eval?

Reply:
Because it's hidden inside the evaluation function and not exposed.  If we find it important enough, I could rework the interfaces.